<launch>

  <node pkg="earth_rover_localization"  name="sbp_arbitrator" type="sbp_arbitrator.py" output="screen">
      <!-- Ntrip params -->
      <param name="ntrip_host" value="rtk2go.com" />
      <param name="ntrip_port" value="2101" />
      <param name="ntrip_mount_point" value="ER_Valldoreix_1" />
      <!-- Radio params -->
      <param name="radio_port" value="/dev/freewaveGXMT14" />
      <param name="radio_baudrate" value="115200" />
        <!-- Udp params -->
      <param name="udp_address" value="192.168.8.222" />
      <param name="udp_port" value="55558" />
    </node>

</launch>
